core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -780,6 +780,41 @@ private SqlLibraryOperators() {
           OperandTypes.STRING_STRING,
           SqlFunctionCategory.TIMEDATE);
 
+  /**
+   * The "TO_TIME(string1, string2)" function; casts string1
+   * to a TIME using the format specified in string2.
+   */
+  @LibraryOperator(libraries = {SNOWFLAKE})
+  public static final SqlFunction TO_TIME =
+      new SqlFunction("TO_TIME",
+          SqlKind.OTHER_FUNCTION,
+          ReturnTypes.DATE_NULLABLE,
+          null,
+          OperandTypes.STRING_STRING,
+          SqlFunctionCategory.TIMEDATE);
+
+  /**
+   * The "TIME(string1)" function; casts string1
+   * to a TIME using the format specified in string2.
+   */
+  @LibraryOperator(libraries = {BIG_QUERY})
+  public static final SqlFunction TIME =
```

Comment:
Check previous comment for TIME and TIMESTAMP as well.

Reply:
corrected